src/lib/containers/discussion_forum/ForumThreadEditor.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Button } from 'react-bootstrap'
+import {
+  useCreateThread,
+  useUpdateThreadTitle,
+  useUpdateThreadMessage,
+} from '../../utils/hooks/SynapseAPI/forum/useForum'
+import { CreateDiscussionThread } from '../../utils/synapseTypes/DiscussionBundle'
+import { MarkdownEditor } from '../markdown/MarkdownEditor'
+
+export type ForumThreadEditorProps = {
+  editTitle?: string
+  editText?: string
```

Comment:
Do you think these are better prop names?
```suggestion
  initialTitle?: string
  initialMessage?: string
```